File: Atcoder/abc056/a.py
a, b = map(str,input().split())
if a == "H" and b == "H":
	ans = "H"
elif a =="H" and b == "D":
	ans = "D"
elif a == "D" and b =="H":
	ans = "D"
elif a =="D" and b =="D":
	ans = "H"
print(ans)

File: Atcoder/abc056/a.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
	char a,b,ans;
	cin>>a>>b;
	if(a=='H'&&b=='D')ans='D';
	if(a=='H'&&b=='H')ans='H';
	if(a=='D'&&b=='H')ans='D';
	if(a=='D'&&b=='D')ans='H';
	cout<<ans<<endl;
}